djangoplicity/newsletters/views.py

```diff
@@ -77,7 +77,7 @@ def get(self, request, *args, **kwargs):
         self.object = self.get_object()
 
         if not self.object.published and not (request.user.is_superuser or request.user.is_staff):
-            if request.user.is_authenticated():
+            if request.user.is_authenticated:
```

Comment:
Ok good, but we need to have this in mind:

request.user.is_authenticated returns a `CallableBool(True)` and fails when checking it like this `if request.user.is_authenticated is True`